spec.emu

```diff
@@ -712,7 +712,7 @@ contributors: Mark S. Miller, Richard Gibson
 
     <emu-clause id="sec-properties-of-the-arraybuffer-instances" number="7">
       <h1>Properties of ArrayBuffer Instances</h1>
-      <p>ArrayBuffer instances inherit properties from the ArrayBuffer prototype object. ArrayBuffer instances each have an [[ArrayBufferData]] internal slot, an [[ArrayBufferByteLength]] internal slot, and an [[ArrayBufferDetachKey]] internal slot. ArrayBuffer instances which are resizable each have an [[ArrayBufferMaxByteLength]] internal slot<ins>, and ArrayBuffer instances which are immutable each have an [[ArrayBufferIsImmutable]] internal slot</ins>.</p>
+      <p>ArrayBuffer instances inherit properties from the ArrayBuffer prototype object. ArrayBuffer instances each have an [[ArrayBufferData]] internal slot, an [[ArrayBufferByteLength]] internal slot, and an [[ArrayBufferDetachKey]] internal slot. ArrayBuffer instances which are resizable each have an [[ArrayBufferMaxByteLength]] internal slot<ins>, and ArrayBuffer instances which are immutable each have an [[ArrayBufferIsImmutable]] internal slot whose value is always *undefined*</ins>.</p>
```

Comment:
```suggestion
      <p>ArrayBuffer instances inherit properties from the ArrayBuffer prototype object. ArrayBuffer instances each have an [[ArrayBufferData]] internal slot, an [[ArrayBufferByteLength]] internal slot, and an [[ArrayBufferDetachKey]] internal slot. ArrayBuffer instances which are resizable each have an [[ArrayBufferMaxByteLength]] internal slot<ins>, and ArrayBuffer instances which are immutable each have an [[ArrayBufferIsImmutable]] internal slot whose value is always ~unused~</ins>.</p>
```
why not this spec value?

Reply:
As noted in the description, alignment with [[[ErrorData]]](https://tc39.es/ecma262/multipage/fundamental-objects.html#sec-properties-of-error-instances) and convenience for [HTML integration](https://github.com/whatwg/html/pull/11033#discussion_r1958969913).

Reply:
fair on ErrorData alignment, although the value of that slot isn't used anywhere currently, and for HTML integration it seems like a JS spec value would work too?

Reply:
I don't know if HTML even has the ability to reference ECMAScript spec values.

Reply:
interesting, i wonder how they handle all the existing places that's a thing. this is fine as-is, ofc.